### PR TITLE
refactor: demo account to oak components - AI-460

### PR DESCRIPTION
**Pull request by JBR90**
## Description

- [This PR](https://github.com/oaknational/oak-ai-lesson-assistant/pull/204) with mobile bugs fixed
- Some requested design changes:

- Demo user modal centered
- fonts and color font aligned with OAK

## Issue(s)

Fixes #

## How to test

1. Go to https://oak-ai-lesson-assistant-c77tv9szu.vercel-preview.thenational.academy
2. Click on \_\_\_\_\_\_
3. You should see \_\_\_\_\_\_

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist

- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Does this PR update a package with a breaking change

### CODE REVIEW

File: apps/nextjs/src/components/AppComponents/Chat/chat-right-hand-side-lesson.tsx [hunk 1]
```diff
@@ -77,7 +77,7 @@ const ChatRightHandSideLesson = ({
           onClick={() => {
             closeMobileLessonPullOut();
           }}
-          className="flex items-center justify-center gap-3"
+          className="mt-22 flex items-center justify-center gap-3 "
```
Comment (codeincontext) [inline apps/nextjs/src/components/AppComponents/Chat/chat-right-hand-side-lesson.tsx:80]: Should this always be 22 or does it depend on the demo status?

**Review by codeincontext — APPROVED** [review]
Code approved, but let's get Mike's test before merging 👍

File: apps/nextjs/src/components/AppComponents/Chat/chat-right-hand-side-lesson.tsx (hunk 1 above)
Comment (JBR90) [reply apps/nextjs/src/components/AppComponents/Chat/chat-right-hand-side-lesson.tsx:80]: Yes should be, will fix this

**Review by mikeritson-oak — CHANGES_REQUESTED** [review]
I've added some notes in the Notion ticket for feedback.

**Comment by JBR90** [comment]
> I've added some notes in the Notion ticket for feedback.



> I've added some notes in the Notion ticket for feedback.

Thanks Mike, I added a couple of comments in the ticket.

**Pull request merged**